File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/RelationAction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace gov.va.medora.mdo.domain.sm
{
    public class RelationAction
    {
        private TriageRelation _triageRelation;

        public TriageRelation TriageRelation
        {
            get { return _triageRelation; }
            set { _triageRelation = value; }
        }
        //private RelationActionEnum _action;

        //public RelationActionEnum Action
        //{
        //    get { return _action; }
        //    set { _action = value; }
        //}
    }
}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/RelationAction.java
package gov.va.med.mhv.sm.model;

import gov.va.med.mhv.sm.enumeration.RelationActionEnum;

public class RelationAction {

	private TriageRelation triageRelation;
	private RelationActionEnum action;
	
	public TriageRelation getTriageRelation() {
		return triageRelation;
	}
	public void setTriageRelation(TriageRelation triageRelation) {
		this.triageRelation = triageRelation;
	}
	public RelationActionEnum getAction() {
		return action;
	}
	public void setAction(RelationActionEnum action) {
		this.action = action;
	}
	
	
	
}
